### PR TITLE
Update theme context package

### PR DESCRIPTION
**Pull request by kavimaluskam**
### Description
- Update theme context package as suggested by [kentcdodds](https://kentcdodds.com/blog/how-to-use-react-context-effectively)
- Update prettier setting: use arrowParens

### QA
- [x] Chrome

### Issuer
- https://github.com/Tremendous-Tiny-Team/ttt-web/issues/3

### TODO
- [ ] Package Theme Module as plug-in or function
- [x] Test function / const performance

### CODE REVIEW

**Comment by kavimaluskam** [comment]
@victor77dev Tested the `function` / `const` implementation does not affect the performance so I am keeping it.

File: src/utils/theme.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import Terser from 'terser';
+
+import {
+  COLOR_MODE_KEY,
+  COLORS,
+  INITIAL_COLOR_MODE_CSS_PROP,
+} from '../styles/themes';
+
+const setColorsByTheme = () => {
+  const colors = '🌈';
```
Comment (victor77dev) [inline src/utils/theme.js:11]: Emoji coding😎

File: .prettierrc [hunk 2]
```diff
@@ -1,5 +1,4 @@
 {
-  "arrowParens": "avoid",
```
Comment (victor77dev) [inline .prettierrc:2]: Better mark this sneak-in prettier format setting in the PR description.

File: src/contexts/ThemeContext.js [hunk 3]
```diff
@@ -58,8 +64,16 @@ export const ThemeProvider = ({ children }) => {
     </ThemeContext.Provider>
   );
 };
+
+const useTheme = () => {
+  const context = useContext(ThemeContext);
+  return context;
```
Comment (victor77dev) [inline src/contexts/ThemeContext.js:70]:
```suggestion
    return useContext(ThemeContext);
```

**Review by victor77dev — CHANGES_REQUESTED** [review]

File: .prettierrc (hunk 2 above)
Comment (kavimaluskam) [reply .prettierrc:2]: oh yes. only included as a single commit but not in PR description.

**Comment by kavimaluskam** [comment]
@victor77dev Sorry just reviewed your response. And things is tested in lab and looks good.

**Review by victor77dev — APPROVED** [review]

**Pull request merged**